components/collective-page/HeroBackground.js

```diff
@@ -39,9 +73,207 @@ const HeroBackground = styled.div`
   }
 `;
 
+const ImageContainer = styled.div`
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  position: absolute;
+  overflow: hidden;
+`;
+
+const BackgroundImage = styled.img.attrs({ alt: '' })`
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  min-height: 0;
+  min-width: 0;
+  max-height: none;
+  max-width: none;
+  position: absolute;
+  margin: auto;
+`;
+
+const KEY_IMG_REMOVE = '__REMOVE__';
+const DEFAULT_CROP = { x: 0, y: 0 };
+
+const getCrop = collective => get(collective.settings, 'collectivePage.background.crop') || DEFAULT_CROP;
+const getZoom = collective => get(collective.settings, 'collectivePage.background.zoom') || 1;
+
+/**
+ * Wraps the logic to display the hero background. Fallsback on a white background if
+ * css `mask` is not supported.
+ */
+const HeroBackground = ({ collective, isEditing, onEditCancel }) => {
+  const [crop, onCropChange] = React.useState(getCrop(collective));
+  const [zoom, onZoomChange] = React.useState(getZoom(collective));
+  const [uploadedImage, setUploadedImage] = React.useState();
+  const [submitting, setSubmitting] = React.useState(false);
+  const hasBackgroundSettings = has(collective.settings, 'collectivePage.background');
+
+  return !isEditing ? (
+    <StyledBackground>
+      {collective.backgroundImage && (
+        <ImageContainer>
+          <BackgroundImage
+            src={collective.backgroundImage}
+            style={
+              hasBackgroundSettings
+                ? { transform: `translate(${crop.x}px, ${crop.y}px) scale(${zoom})` }
+                : { minWidth: '100%' }
+            }
+          />
+        </ImageContainer>
+      )}
+    </StyledBackground>
+  ) : (
+    <Mutation mutation={EditCollectiveBackgroundMutation}>
+      {editBackground => (
+        <StyledBackground backgroundImage={collective.backgroundImage} isEditing>
+          <Cropper
+            image={uploadedImage ? uploadedImage.preview : collective.backgroundImage}
+            cropSize={{ width: BASE_WIDTH, height: BASE_HEIGHT }}
+            apect={BASE_WIDTH / BASE_HEIGHT}
```

Comment:
Here it should be `aspect`. Anyway, if you use `cropSize`, `aspect` is optional :)

Reply:
Good catch! I'll make a PR to update this